tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -81,11 +83,34 @@ <h3>Profiling isn’t supported in Colab yet.</h3>
                always-float-label
                type="number"
                min=1
+               max=600000
+               auto-validate
+               pattern="[0-9]+"
+               error-message="Input is not an integer between 1 to 600000"
+               required
                value="{{_profileDuration}}"></paper-input>
+  <div hidden="{{!_shouldShowTpuOptions(_profilerServiceAddressType)}}">
+    <paper-checkbox checked="{{_profileIncludeDatasetOps}}">Trace dataset ops</paper-checkbox>
```

Comment:
What are the differences between dataset ops and other cpu activities? Why do we need to turn this on explicitly for TPU? I think for GPU, all dataset ops will be traced once turn on cpu profiling.

Reply:
The dataset ops was found to be dominant in the CPU traces, removing that allows us to see more TPU device events. As you said, the filtering was not done for GPU, therefore I set it as an TPU only option.